### ISSUE TITLE
Kitty to Darknet issue

### ISSUE THREAD

**Issue by DanIn232** [monocongo/cvdata#149]
Hi,

Thank you for writing this very useful piece of code.  I have been trying to convert Kitty annotations to Darknet ones, and I'm having an issue with it. this is the input:

cvdata_convert --in_format kitti --out_format darknet --annotations_dir old --images_dir old --out_dir new  

error: 
TypeError: join() argument must be str or bytes, not 'NoneType'

Any idea what's wrong?

Thanks

**Comment by monocongo** [comment]
Yes, it looks like the KITTI files are missing fields (column values) in one or more of the records (rows), and so when it's trying to perform a join it's failing because the missing field(s) result in a `None`, and you can't join to a `None` (or NoneType). That's one conjecture, anyway.  :)

If you can provide your input file(s) and as much about what you're trying to do here I may be able to get to the specific issue where this is happening, otherwise I can only speculate. Also, I apologize in advance because I don't have too much time to devote to this project currently, so it may take a little while...
